Samples/Windowing/cs-winforms-unpackaged/AppForm.cs

```diff
@@ -452,5 +508,40 @@ private void resetTitlebarBtn_Click(object sender, EventArgs e)
             _mainAppWindow.Title = WindowTitle;
             MyTitleBar.Visible = false;
         }
+
+        private void StandardHeight_Click(object sender, EventArgs e)
+        {
+            _isTallTitleBar = false;
+            toggleTitlebarHeightOption();
+        }
+        private void TallHeight_Click(object sender, EventArgs e)
+        {
+            _isTallTitleBar = true;
+            toggleTitlebarHeightOption();
+        }
+        private void toggleTitlebarHeightOption()
```

Comment:
Same comment as for WinUI-C# sample: these should probably be folded into a single method (keep the toggleTitlebarHeightOption) to simplify and make sure the _isTallTitleBar variable is not out of sync with reality.